### PR TITLE
Add contribution links within context #1000 #1001

### PR DESCRIPTION
**Pull request by Tosin-Balogun**
## Description

This is a prototype based on improving the users access to our contribution process through the service manual website

We will use this pull request to review changes, updates et al
### Related issue

#1000 #1001 
## Checklist

- [x] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ ] CHANGELOG entry
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry
- [ ] Page updated date

### CODE REVIEW

File: app/views/includes/_contact-panel.njk [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% if pageSection === "Content style guide" %}
 
 <p> Please let us know how this has worked for you. This will help us improve it for everyone.</p>
-<p>Before you start, you will need a <a href="https://github.com/join?source=login">GitHub account</a>. It's an open forum where we collect feedback</p>
+<p>Before you start, you will need a <a href="https://github.com/join?source=login">GitHub account</a>. It's an open forum where we collect feedback.</p>
```
Comment (Tosin-Balogun) [inline app/views/includes/_contact-panel.njk]: @sarawilcox - updated the text

File: app/views/includes/_contact-panel.njk [hunk 2]
```diff
@@ -16,7 +16,7 @@
       <path d="M0 0h24v24H0z" fill="none"></path>
       <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
     </svg>
-    <span class="nhsuk-action-link__text">Share your research findings </span>
+    <span class="nhsuk-action-link__text">Share your research findings on GitHub</span>
```
Comment (Tosin-Balogun) [inline app/views/includes/_contact-panel.njk:19]: @sarawilcox - updated action link

File: app/views/includes/_contact-panel.njk (hunk 1 above)
Comment (Tosin-Balogun) [reply app/views/includes/_contact-panel.njk:11]: @sarawilcox - updated text with new suggested content

**Review by sarawilcox — APPROVED** [review]
LGTM Tosin.

**Comment by Tosin-Balogun** [comment]
@sarawilcox - hold on, i'm currently adding the crosslinks to the backlog issue on each page

**Pull request merged**